LINUX/netmap_linux.c

```diff
@@ -1824,8 +1824,11 @@ nm_os_pt_memdev_iomap(struct ptnetmap_memdev *ptn_dev, vm_paddr_t *nm_paddr,
 	*mem_size = ioread32(ptn_dev->pci_io + PTNET_MDEV_IO_MEMSIZE_HI);
 	*mem_size = ioread32(ptn_dev->pci_io + PTNET_MDEV_IO_MEMSIZE_LO) |
 		(*mem_size << 32);
-
+#ifdef __arm__
+	nm_prinf("=== BAR %d start %x len %x mem_size %lx ===",
+#else
```

Comment:
Instead of this, would it work to explicitely cast the return value of pci_resource_start and pci_resource_len to (unsigned long long)?

Reply:
Well, if I well understood this is used by ptnet that is related to QEMU/KVM. 
I currently don't have a setup to test it, but I think that casting the two returns to unsigned long long it will work as well.
As it removes compiler warnings as well, so up to me it is more than fine.